ui/selectmenu.js

```diff
@@ -498,6 +501,7 @@ return $.widget( "ui.selectmenu", {
 		// Change native select element
 		this.element[ 0 ].selectedIndex = item.index;
 		this.buttonItem.replaceWith( this.buttonItem = this._renderButtonItem( item ) );
+		this._addClass( this.buttonItem, "ui-selectmenu-text" );
```

Comment:
As far as i see though these are not ever in the same call chain and since its a single statement i don't see a reason to make it its own method.